scripts/tests/happy_tests.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+#
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This file is used to test a single happy test using (almost) the same
+# method as ninja does. Suggest passing $HAPPY_LOG_DIR to set path for logs.
+
+SOURCE="${BASH_SOURCE[0]}"
+SOURCE_DIR="$(cd "$(dirname "$SOURCE")" >/dev/null 2>&1 && pwd)"
+REPO_DIR="$SOURCE_DIR/../../"
+
+export HAPPY_LOG_DIR="${HAPPY_LOG_DIR:-$(mktemp -d)}"
+
+set -x
+env
+
+function happytest_bootstrap() {
+    echo "Bootstrapping Happy Test"
+    set -e
+    # Bootstrap Happy
+    cd "$REPO_DIR/third_party/happy/repo"
+
+    # Override happy log dir config.
+    python3 -c 'import json, os; file = open("happy/conf/main_config.json"); data = json.load(file); data["log_directory"] = os.environ["HAPPY_LOG_DIR"]; out = open("happy/conf/main_config.json", "w"); json.dump(data, out);'
+
+    apt-get update
+    apt-get install -y bridge-utils \
```

Comment:
agree, this need to put in dockerfile, push it to the docerhub

Reply:
Could you create a PR to update docker (version and packages)?
I can handle the push once a PR is submitted (and even a temporary build to test before submitting, since version will be bumped)

Reply:
For now, we decided to use separate job for tests, since we don't want to introduce sudos and privileged containers in main build.

Reply:
the host machine in github action don't have ipv6 support. we need to run it inside docker with ipv6 configuration. happy orchestrate lightweight ip netns, which need privildege support in docker. 

Agree. please consider to add these apt-get packages in docker as @andreilitvin suggesed.

Reply:
Something like this: https://github.com/project-chip/connectedhomeip/pull/2854